modules/extensions-spring-support/src/test/java/org/mule/test/module/extension/validation/InvalidExtensionConfigTestCase.java

```diff
@@ -0,0 +1,183 @@
+  }
+
+  @Test
+  public void petStoreExclusiveGroupPojo() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-group-pojo-config.xml:8]: "
+            + "Element <config>, the following parameters cannot be set at the same time: [cash, debt].");
+    loadConfiguration("validation/petstore-exclusive-group-pojo-config.xml");
+  }
+
+  @Test
+  public void petStoreExclusiveGroupInsidePojo() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-group-inside-pojo-config.xml:13]: "
+            + "Element <Aquarium>, the following parameters cannot be set at the same time: [frogName, fishName].");
+    loadConfiguration("validation/petstore-exclusive-group-inside-pojo-config.xml");
+  }
+
+  @Test
+  public void petStoreExclusiveParameterOperation() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-parameters-operation.xml:16]: "
+            + "Element <getBreeder>, the following parameters cannot be set at the same time: [mammals, birds].");
+    loadConfiguration("validation/petstore-exclusive-parameters-operation.xml");
+  }
+
+  @Test
+  public void petStoreExclusiveParameterWithAlias() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-parameter-with-alias.xml:8]: "
+            + "Element <config>, the following parameters cannot be set at the same time: [cash, debt].");
+    loadConfiguration("validation/petstore-exclusive-parameter-with-alias.xml");
+  }
+
+  @Test
+  public void petStoreExclusiveGroupInsidePojoOneRequired() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-group-inside-pojo-one-required-config.xml:13]: "
+            + "Element <Aquarium> requires that one of its optional parameters must be set, but all of them are missing. One of the following must be set: [frogName, fishName].");
+    loadConfiguration("validation/petstore-exclusive-group-inside-pojo-one-required-config.xml");
+  }
+
+  @Test
+  public void petStoreExclusiveParameterRequired() throws Exception {
+    expectedException.expect(ConfigurationException.class);
+    expectedException
+        .expectMessage("[validation/petstore-exclusive-required-parameter.xml:9]: "
+            + "Element <Breeder> requires that one of its optional parameters must be set, but all of them are missing. One of the following must be set: [mammals, birds].");
+    loadConfiguration("validation/petstore-exclusive-required-parameter.xml");
+  }
+
+  // TODO MULE-19350 migrate and adapt this test
```

Comment:
use appropriate comment format for a method or add this this comment in the method.

Reply:
this is just a TODO, not a description of the test